docs/voice/telephony-session-notifications.md

```diff
@@ -0,0 +1,225 @@
+# Telephony Session Notification
+
+A developer can use Telephony Session Notification to detect all changes in the status of a call effectively. 
+
+## What is Telephony Session Notification?
+
+A Telephony Session Notification (Telephony Session Notification) is a series of notifications or events a developer can use to track an end-to-end telephony call in RingCentral. The call can be from an external PSTN or any mobile number to a RingCentral extension, or it can be from a RingCentral Digital Line to an external PSTN number, or it can be between two RingCentral extensions. The Telephony Session Notification provides detailed end-to-end call sessions and party (endpoints/user) level information to enable developers to take specific actions or build custom reports based on that call information. The Telephony Session Notification events can be subscribed both at an Account level or at an Extension level.
+
+Account Level Telephony Session Notification -  Will track and provide notifications for all the extensions within an account.
+
+Extension Level Telephony Session Notification - Will track calls on a specific extension.
+
+Detailed instructions on the [Notification types](https://developers.ringcentral.com/api-reference/Account-Presence-Event) are available in the API reference.
+
+Detailed instructions on how to subscribe to [RingCentral events](https://developers.ringcentral.com/guide/notifications) or notifications are available in the API reference
+
+
+## What are the use cases?
+
+<h5>Call Analytics</h5> -  Developers can use the raw data from Telephony Session Notification events and build call analytics dashboards replicating live reports which use the same event streams. Some of the standard call analytics metrics and their corresponding formulas are:
```

Comment:
Please remove HTML based `<h5>` formatting and replace with Markdown. See below for GitHub Markdown syntax.

https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax